### DISCUSSION TITLE
api-2.0 degradation 1.0 failed

### DISCUSSION THREAD

**Discussion by ouyoung33** [jaredhendrickson13/pfsense-api#436] [Q&A]
The previous installation was api-2.0, there was no way to delete it directly with the command (not recognized), I restored pfsense to the appearance Settings, and then reinstalled api-1.0 still reported an error
![Uploading image.png…]()

**Comment by ouyoung33** [comment]
The error message is:Fetching pfSense-2.7-pkg-API.pkg: 100%    2 MiB   1.7MB/s    00:01
Installing pfSense-pkg-API-1.7...
pkg: pfSense-pkg-API-1.7 conflicts with pfSense-pkg-RESTAPI-2.0_0-dev-cbc8ea0 (installs files into the same place).  Problematic file: /etc/inc/firebase/php-jwt/README.md

Failed to install the following 1 package(s): https://github.com/jaredhendrickson13/pfsense-api/releases/latest/download/pfSense-2.7-pkg-API.pkg

**Comment by jaredhendrickson13** [comment]
Duplicate #434
